src/odemis/util/polar.py

```diff
@@ -341,24 +427,107 @@ def _CreateMirrorMask(data, pixel_size, pole_pos, hole=True):
     X, Y = data.shape
     pole_x, pole_y = pole_pos
 
+    # create mirror mask in cartesian coordinates
+
     # Calculate the coordinates of the cutoff of half circle
     center_x = pole_x
     lower_y = pole_y - ((2 * parabola_f - focus_distance) / pixel_size[1])
     center_y = pole_y - ((2 * parabola_f) / pixel_size[1])
 
-    # Compute the radius
-    r = (2 * math.sqrt(xmax * parabola_f)) / pixel_size[1]
+    # Compute the dilated radius
+    # use dilated mask to handle edge effects for triangulation code (offset_radius)
+    r = (2 * math.sqrt(xmax * parabola_f)) / pixel_size[1] + offset_radius
     y, x = numpy.ogrid[-center_y:X - center_y, -center_x:Y - center_x]
     circle_mask = x * x + y * y <= r * r
 
     # Create half circle mask
-    circle_mask[:int(lower_y), :] = False
+    # check that center of mask is located within image (e.g. if mask at the edge of image not cutoff)
+    if (lower_y - offset_radius) > 0:
+        circle_mask[:int(lower_y) - offset_radius, :] = False
 
     # Crop the pole making hole of hole_diameter
+    # For delaunay triangulation hole=False: to avoid edge effects
     if hole:
-        r = (hole_diameter / 2) / pixel_size[1]
+        r_hole = (hole_diameter / 2) / pixel_size[1]
         y, x = numpy.ogrid[-pole_y:X - pole_y, -pole_x:Y - pole_x]
-        circle_mask_hole = x * x + y * y <= r * r
+        circle_mask_hole = x * x + y * y <= r_hole * r_hole
         circle_mask = numpy.where(circle_mask_hole, 0, circle_mask)
 
+    ###########################################
```

Comment:
Let's not put the complete history of the code in comments. Please remove this whole part, and keep it somewhere else (eg, Dropbox) if you think it's code that could be useful some time in the future.